composer.json

```diff
@@ -13,13 +13,9 @@
     },
     "scripts": {
         "install-bats": "if [ ! -f bin/bats ] ; then git clone https://github.com/sstephenson/bats.git; mkdir -p $HOME/bin; bats/install.sh $HOME; fi",
-        "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
-        "bats": "TERMINUS_PLUGINS_DIR=.. PATH=$HOME/bin:$PATH bats -p -t tests",
+        "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer",
+        "bats": "PATH=$HOME/bin:$PATH bats -p -t tests",
```

Comment:
I'm thinking calling `$HOME/bin/bats` directly without setting the `PATH` is a bit cleaner.  Also, would `COMPOSER_BIN_DIR=$HOME/bin` make sense?

Reply:
it just needs `bats` to be in PATH env var. It's modifying $PATH to add $HOME/bin for the situation where the binary was put there (docker images I guess?) But I think you're right and this is not needed. I'll try removing it

Reply:
I simplified this a bit. If tests passes, then it works :)